### PR TITLE
feat: [PL-21605]: Making errors in K8 connector consistent

### PR DESCRIPTION
**Pull request by avinashmadhwani02**
#### Screenshots

Before
Screenshot 2023-03-07 at 5 48 31 PM

After
Screenshot 2023-03-07 at 5 40 01 PM

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by avinashmadhwani02** [comment]
retrigger titlecheck

**Review by abhinavrastogi-harness — APPROVED** [review]

File: src/modules/35-connectors/components/CreateConnector/K8sConnector/StepAuth/Stepk8ClusterDetails.tsx [hunk 1]
```diff
@@ -297,7 +297,7 @@ const Stepk8ClusterDetails: React.FC<StepProps<Stepk8ClusterDetailsProps> & K8Cl
         is: delegateType => delegateType === DelegateTypes.DELEGATE_OUT_CLUSTER,
         then: Yup.string().required(getString('validation.authType'))
       }),
-    username: Yup.string()
+    usernametextField: Yup.string()
```
Comment (mayankVermaHarness) [inline src/modules/35-connectors/components/CreateConnector/K8sConnector/StepAuth/Stepk8ClusterDetails.tsx:300]: I don't see any other reference for `usernametextField`. Is this intended?

Comment (avinashmadhwani02) [reply src/modules/35-connectors/components/CreateConnector/K8sConnector/StepAuth/Stepk8ClusterDetails.tsx:300]: <TextReference /> has usernametextField
there's no reference for username as well

**Review by mayankVermaHarness — APPROVED** [review]

File: src/modules/35-connectors/components/CreateConnector/K8sConnector/StepAuth/Stepk8ClusterDetails.tsx (hunk 1 above)
Comment (mayankVermaHarness) [reply src/modules/35-connectors/components/CreateConnector/K8sConnector/StepAuth/Stepk8ClusterDetails.tsx:300]: Got it. `<TextReference />` internally uses string interpolation

**Review by abhinavrastogi-harness — APPROVED** [review]

**Pull request merged**